libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -2187,6 +2186,12 @@ double WaveTrack::GetEndTime() const
    return ChannelGroup::GetEndTime();
 }
 
+double WaveTrack::SnapToSample(double t) const
+{
+   const auto sampleRate = GetRate();
+   return std::round(t * sampleRate) / sampleRate;
```

Comment:
I think this function should use `TimeToLongSamples`

Reply:
No, just delete the function!  It is defined but not used anywhere in these six PRs

Reply:
It is used, in `WaveTrack::ApplyStretchRatio`, in PR 5. I'll remove it from this PR, and rebasing will force me to re-introduce it. Then I'll use `TimeToLongSamples`.